src/VSCodeExtension/package.json.v.template

```diff
@@ -137,15 +247,15 @@
     },
     "blobs": {
         "win32": {
-            "url": "https://msquantumpublic.blob.core.windows.net/qsharp-compiler/LanguageServer-win10-x64-#SEMVER_VERSION#.zip",
+            "url": "https://msquantumpublic.blob.core.windows.net/qsharp-compiler/LanguageServer-win10-x64-0.9999.22060152.zip",
```

Comment:
This was most likely changed by mistake. Make sure that the version number variable is used instead.